src/app/(with-header)/mypage/reservations/components/ReviewModal.tsx

```
@@ -0,0 +1,141 @@
+    <Modal isOpen={isOpen} onOpenChange={(open) => !open && handleClose()}>
+      <Modal.Content className='!h-750 !w-480 !max-w-none !min-w-0 !rounded-xl !p-0'>
+        <div className='flex h-full w-full flex-col overflow-hidden rounded-xl bg-white shadow-lg'>
+          {/* 헤더 */}
+          <div className='flex items-center justify-between px-24 pt-23 pb-41'>
+            <h2 className='text-nomad text-xl font-bold'>후기 작성</h2>
+            <button
+              onClick={handleClose}
+              className='text-gray-400 hover:text-gray-600'
+              disabled={isLoading}
+            >
+              <Close size={24} color='currentColor' />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)